src/compiler/reviews.md

```diff
@@ -1,104 +1,81 @@
-Review Policy
-=============
+# Review Policy
```

Comment:
changes in this file are also intertwined with reformatting and it's too much for me to review...

Reply:
I don't think I made any changes to this particular file, just unified the formatting with our other pages.